sdk/communication/communication-short-codes/tsconfig.json

```diff
@@ -5,7 +5,8 @@
     "declarationDir": "./types",
     "paths": {
       "@azure-tools/communication-short-codes": ["./src/index"]
-    }
+    },
+    "lib": ["dom"]
```

Comment:
If you do the `/// reference` thing I've mentioned a few times, you shouldn't need this. There is no difference between the two (or at least, shouldn't be), so feel free to ignore my `/// reference` comments if you prefer to just use this.